README.md

```diff
@@ -18,23 +18,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: '1.10'                      # replace this with whatever version you need
```

Comment:
Suggest 1 instead of 1.10? Most users would be good with that, the few who need a specific version know what to do (and it's hinted in the comment)

Reply:
```suggestion
          version: '1'                         # replace this with whatever version you need
```